flake.nix

```diff
@@ -39,15 +40,22 @@
       (flake-utils.lib.eachSystem [ "x86_64-linux" "x86_64-darwin" ]
         (system:
         let
+          pkgs = import inputs.nixpkgs { inherit system; };
           hydraProject = import ./nix/hydra/project.nix {
-            inherit (inputs) nixpkgs haskellNix iohk-nix CHaP;
-            inherit system;
+            inherit (inputs) haskellNix iohk-nix CHaP;
+            inherit system nixpkgs;
           };
-        in
-        {
-          packages = import ./nix/hydra/packages.nix {
+          hydraPackages = import ./nix/hydra/packages.nix {
             inherit hydraProject system;
           };
+          hydraImages = import ./nix/hydra/docker.nix {
+            inherit hydraPackages system nixpkgs;
+          };
+        in
+        rec {
+          packages = hydraPackages // {
```

Comment:
what does `//` means?

is it like the set difference operation?

Reply:
It's the opposite, it's a union and thus merges the argument set on the right hand side to the one on the left hand side.